code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,9 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    stylePreference: {
+      type: DataTypes.TEXT
```

Comment:
We chose TEXT here to hold the string or potential array of strings if a user has multiple subscriptions.